### PR TITLE
Adds iOS integration test for access denied scenario

### PR DESCRIPTION
**Pull request by crumpf**

### CODE REVIEW

File: iosApp/iosAppTests/Support/TestContentController.swift [hunk 1]
```diff
@@ -180,9 +190,14 @@ class TestContentController: MessengerHandler {
...
 
-    func waitForExpectation() {
-        let result = XCTWaiter().wait(for: [testExpectation!], timeout: 60)
-        XCTAssertEqual(result, .completed, "Expectation never fullfilled: \(testExpectation?.description ?? "No description.")")
+    func waitForTestExpectation(timeout: Double = 60.0) {
+        let result = XCTWaiter().wait(for: [testExpectation!], timeout: timeout)
+        XCTAssertEqual(result, .completed, "Test expectation never fullfilled: \(testExpectation?.description ?? "No description.")")
+    }
+    
+    func waitForErrorExpectatio(timeout: Double = 60.0) {
```
Comment (AfanasievAnton) [inline iosApp/iosAppTests/Support/TestContentController.swift:198]: typo: missing `n` at the end of `Expectation`

File: iosApp/iosAppTests/Support/TestContentController.swift [hunk 2]
```diff
@@ -180,9 +190,14 @@ class TestContentController: MessengerHandler {
...
+        let result = XCTWaiter().wait(for: [errorExpectation!], timeout: timeout)
```
Comment (mmorehou) [inline iosApp/iosAppTests/Support/TestContentController.swift:199]: You could set a guard before this line to make sure we have an errorExpectation available before running this.

**Review by mmorehou — APPROVED** [review]
Nice work!

File: iosApp/iosApp.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 51;
+	objectVersion = 50;
```
[the remark this answers is not in the document]
Comment (crumpf) [reply iosApp/iosApp.xcodeproj/project.pbxproj:6]: Yeah, weird. I didn't make any explicit project changes. The `objectVersion`, I believe, is related to the "Project Format". I'll update the project format to "Xcode 10.0-compatible" which will move the `objectVersion` back to `51`.

**Review by AfanasievAnton — APPROVED** [review]

**Comment by crumpf** [comment]
Jenkins had a successful run.

**Pull request merged**